### DISCUSSION TITLE
Diary-中国华侨历史博物馆 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#339] [Announcements]
# Diary-中国华侨历史博物馆 | Zi-Zi's Journey

来了解一下“润佬”的历史。

https://gz-metal-cell.github.io/2024/07/08/Diary-%E4%B8%AD%E5%9B%BD%E5%8D%8E%E4%BE%A8%E5%8E%86%E5%8F%B2%E5%8D%9A%E7%89%A9%E9%A6%86/

**Comment by promefire** [comment]
图片的排列太优雅了吧😭

**Reply by GZ-Metal-Cell** [reply]
没有 pro 神优雅😭
